app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/BonusSectionView.kt

```diff
@@ -0,0 +1,73 @@
+package com.aptoide.android.aptoidegames.feature_apps.presentation
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.StrokeJoin
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.unit.dp
+import cm.aptoide.pt.extensions.PreviewDark
+import com.aptoide.android.aptoidegames.drawables.icons.getPromoSection
+import com.aptoide.android.aptoidegames.theme.AGTypography
+import com.aptoide.android.aptoidegames.theme.Palette
+
+@Composable
+fun BonusSectionView() {
+  Box(
+    modifier = Modifier.padding(vertical = 24.dp)
+  ) {
+    Image(
+      imageVector = getPromoSection(
+        iconColor = Palette.Primary,
+        outlineColor = Palette.Black,
+        backgroundColor = Palette.Secondary,
+        themeColor = Palette.Black
+      ),
+      contentDescription = "Bonus Section",
+      modifier = Modifier
+        .fillMaxWidth(),
+      contentScale = ContentScale.FillWidth,
+    )
+    Column(modifier = Modifier.padding(start = 16.dp, top = 44.dp)) {
+      Box {
```

Comment:
Maybe this should be a reusable composable component where we pass the text and the configurable style